index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
+
```

Comment:
Kindly use semantic HTML tags as it makes the HTML more comprehensible by better defining the different sections and layout of web pages. That allows browsers and search engines to better interpret the content.Think of changing the <div> here with the <main> tag.